package.json

```diff
@@ -61,8 +61,7 @@
     "rollup-plugin-copy": "~3.4.0",
     "rollup-plugin-delete": "~1.2.0",
     "rollup-plugin-generate-html-template": "~1.7.0",
-    "rollup-plugin-postcss": "~2.0.3",
-    "rollup-plugin-scss": "~2.1.0",
```

Comment:
The PR is about `rollup-plugin-postcss` and `rollup-plugin-styles`, why did you remove the `rollup-plugin-scss` too? Just to give some context.

Reply:
Because it was not used 🤣, I don't know what it was doing there